library/network/src/modules/firewalld_wrapper.rb

```diff
@@ -30,29 +30,70 @@ module Yast
   # mainly a firewalld wrapper. It is inteded to be used mostly by YaST
   # modules written in Perl like yast-dns-server.
   class FirewalldWrapperClass < Module
+    include Logger
+
+    VALID_PROTOCOLS = ["udp", "tcp"].freeze
```

Comment:
do we need to stick with these protocols only? (note: I came from telecomunications where we used e.g. SCTP quite often ;-)

Reply:
probably we will support also sctp & dccp but current needs are TPC and UDP ...

Reply:
added both